src/utility.js

```diff
@@ -53,14 +53,15 @@ function parseUncssrc(filename) {
 function parsePaths(source, stylesheets, options) {
     return stylesheets.map(function (sheet) {
         var sourceProtocol;
+        var isLocalFile = sheet.substr(0, 5) === 'file:';
 
         if (sheet.substr(0, 4) === 'http') {
             /* No need to parse, it's already a valid path */
             return sheet;
         }
 
         /* Check if we are fetching over http(s) */
-        if (isURL(source)) {
+        if (isURL(source) && !isLocalFile) {
```

Comment:
...such that this is really only used for URLs without protocols (path-, host-, and protocol-relative urls). And maybe a comment about that too.